cmd/config.go

```diff
@@ -63,24 +70,35 @@ func init() {
 }
 
 func enableAsPlugin(execPath, execFile string) {
+
 	var found bool
 	color.Cyan("Installing tasty as kubectl and oc CLI plugin")
 
 	execOcLink := execPath + "/oc-olm"
 	err := os.Symlink(execFile, execOcLink)
 	if err != nil {
-		color.Yellow("Oc Plugin already installed.")
+		if strings.Contains(err.Error(), "file exists") {
+			found = true
+		} else {
+			log.Fatal(err)
+		}
+	} else {
 		found = true
```

Comment:
found shouldn't be set to True when your code actually creates the link

Reply:
If found is not set to true the log messafe of Already installed is not shown

Reply:
```
$ which oc-olm
/xxx/bin/oc-olm
$  which kubectl-olm
/xxx/bin/kubectl-olm
$  rm /xxx/bin/kubectl-olm
$ tasty config -p
Installing tasty as kubectl and oc CLI plugin
Tasty already installed as oc and kubectl plugin, you can try oc olm --help or kubeclt olm --help
```

Reply:
the thing is that if we want to say something when is already installed and when de installation has finished successfully, but if we remove the found true and is the first time that is installed will not show any message to the user

Reply:
check it out with the new changes